Modules/Filtering/ImageGrid/test/itkWarpImageFilterTest.cxx

```diff
@@ -372,39 +372,21 @@ itkWarpImageFilterTest(int, char *[])
     ++streamIter;
   }
 
-
-  if (!testPassed)
-  {
-    std::cout << "Test failed." << std::endl;
-    return EXIT_FAILURE;
-  }
-
   // Exercise error handling
 
-
   using InterpolatorType = WarperType::InterpolatorType;
   InterpolatorType::Pointer interp = warper->GetModifiableInterpolator();
-  try
-  {
-    std::cout << "Setting interpolator to nullptr" << std::endl;
-    testPassed = false;
-    warper->SetInterpolator(nullptr);
-    warper->Update();
-  }
-  catch (const itk::ExceptionObject & err)
-  {
-    std::cout << err << std::endl;
-    testPassed = true;
-    warper->ResetPipeline();
-    warper->SetInterpolator(interp);
-  }
 
-  if (!testPassed)
-  {
-    std::cout << "Test failed" << std::endl;
-    return EXIT_FAILURE;
-  }
+  std::cout << "Setting interpolator to nullptr" << std::endl;
+  warper->SetInterpolator(nullptr);
+
+  ITK_TRY_EXPECT_EXCEPTION(warper->Update());
+
+
+  warper->ResetPipeline();
+  warper->SetInterpolator(interp);
+
 
   std::cout << "Test passed." << std::endl;
-  return EXIT_SUCCESS;
+  return testPassed;
```

Comment:
wrong, _testPassed_ is type _bool_

Reply:
Had spotted that. Thanks for the heads-up. Fixed in a36b621.